### DISCUSSION TITLE
UCB EE120 : Signal and Systems - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#166] [Announcements]
# UCB EE120 : Signal and Systems - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%94%B5%E5%AD%90%E5%9F%BA%E7%A1%80/signal/

**Comment by ThaumielFanGirl** [comment]
课程视频好像没了

**Reply by qiuqiumaomao** [reply]
要是一个个都会删除。。那就得提前备份好多了

**Comment by ysh722** [comment]
请问还有课程视频吗，网站上好像找不到了

**Reply by fedoraiver** [reply]
同求

**Comment by s1xs1x** [comment]
蹲蹲

**Reply by fedoraiver** [reply]
https://www.bilibili.com/video/BV1k341187vL?p=22&vd_source=0d6ea723ee5c03ce9136ada767c6889b

**Comment by dghtucs** [comment]
建了一个Q群，讨论可入745931952

**Comment by cy-Yin** [comment]
这里推一下我写的 19fall EE120 的 6 个Lab 的 solution：
[UCBerkeley-EE120-Lab-Solutions](https://github.com/cy-Yin/UCBerkeley-EE120-Lab-Solutions)
代码加了一些注释，回答得也更全面一些

**Reply by jiangruixin** [reply]
请问你有hw和lab的资源吗

**Reply by cy-Yin** [reply]
6个lab的资源我上传到repo的releases里面了

**Comment by WangYuanze111** [comment]
网站登不上去了嘛

**Reply by xiaoxietongxue** [reply]
我也登不上去了

**Comment by jahox0102** [comment]
b站有翻译的视频：https://www.bilibili.com/video/BV1J7421o7pF/?spm_id_from=333.788&vd_source=31309098ed33108a2117a3a49df3212f

**Comment by JFKTRSDAKIDHIKA** [comment]
23fall是一个印度人讲的，讲的很细很慢，手写板书，公式一条条推导，lecture还不错的。

**Comment by ZQH-21** [comment]
请问有无lab和hw的整合资源下载，网站现在进不去了，wayback machine看的lab没有对应

**Reply by JFKTRSDAKIDHIKA** [reply]
wayback machine能提供下吗

**Reply by jiangruixin** [reply]
有了吗

**Reply by ukeSJTU** [reply]
给所有后来到这个课程的人，我下载了所有的课程材料并且上传到github了：https://github.com/ukeSJTU/EE120/tree/main

**Comment by ukeSJTU** [comment]
给所有后来到这个课程的人，我下载了所有的课程材料并且上传到github了：https://github.com/ukeSJTU/EE120/tree/main

**Reply by PeterZhang893** [reply]
感谢您

**Reply by Rishi-Mishima** [reply]
太感谢了！

**Reply by requiemk0612** [reply]
万分感谢
